### ISSUE TITLE
RtGistCommentTest.java:57-59: Apply this rule to all...

### ISSUE THREAD

**Issue by 0pdd** [jcabi/jcabi-github#1388]
The puzzle `1377-59643c80` from #1377 has to be resolved:

https://github.com/jcabi/jcabi-github/blob/1e903e59a027304a2c8270182a7f5c0687fc3d25/src/test/java/com/jcabi/github/RtGistCommentTest.java#L57-L59

The puzzle was created by George Aristy on 14-Apr-18. 

Estimate: 30 minutes, role: DEV.

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@amihaiemil[/z](https://www.0crat.com/u/amihaiemil) please, pay attention to this issue

**Comment by 0crat** [comment]
@0pdd[/z](https://www.0crat.com/u/0pdd) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C9R53K5JA) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by 0crat** [comment]
Job [#1388](https://github.com/jcabi/jcabi-github/issues/1388) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@da044789-3a52-46a3-9276-7cb0d37b26bd, thread: BkParallel-6
com.zerocracy.radars.github.ReSafe[76] java.io.IOException: java.lang.IllegalStateException: Duplicate claims are not allowed in [CA2075Q49](https://www.0crat.com/p/CA2075Q49), can't add this XML:
<?xml version="1.0" encoding="UTF-8"?>
<claim id="1041819285">
   <created>2018-04-14T18:15:52.539Z</created>
   <type>Cancel order</type>
   <token>github;sunsofterp/portal;12;381349045</token>
   <author>g4s8</author>
   <params>
      <param name="job">gh:sunsofterp/portal#12</param>
      <param name="repo">sunsofterp/portal</param>
      <param name="issue">12</param>
      <param name="comment">381349045</param>
   </params>
</claim>

com.zerocracy.pm.Claims[111] java.lang.IllegalStateException: Duplicate claims are not allowed in [CA2075Q49](https://www.0crat.com/p/CA2075Q49), can't add this XML:
<?xml version="1.0" encoding="UTF-8"?>
<claim id="1041819285">
   <created>2018-04-14T18:15:52.539Z</created>
   <type>Cancel order</type>
   <token>github;sunsofterp/portal;12;381349045</token>
   <author>g4s8</author>
   <params>
      <param name="job">gh:sunsofterp/portal#12</param>
      <param name="repo">sunsofterp/portal</param>
      <param name="issue">12</param>
      <param name="comment">381349045</param>
   </params>
</claim>

```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm)

**Comment by 0crat** [comment]
The job [#1388](https://github.com/jcabi/jcabi-github/issues/1388) assigned to @llorllale[/z](https://www.0crat.com/u/llorllale), here is [why](https://www.0crat.com/footprint/C9R53K5JA/1082049776); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html)

**Comment by 0pdd** [comment]
The puzzle `1377-59643c80` has disappeared from the source code, that's why I closed this issue.

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1394](https://github.com/jcabi/jcabi-github/issues/1394) is still not solved.

**Comment by 0crat** [comment]
Order was finished: +30 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
The job [#1388](https://github.com/jcabi/jcabi-github/issues/1388) is now out of scope

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1399](https://github.com/jcabi/jcabi-github/issues/1399) is still not solved; solved: [#1394](https://github.com/jcabi/jcabi-github/issues/1394).

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1404](https://github.com/jcabi/jcabi-github/issues/1404) is still not solved; solved: [#1394](https://github.com/jcabi/jcabi-github/issues/1394), [#1399](https://github.com/jcabi/jcabi-github/issues/1399).

**Comment by 0pdd** [comment]
@0pdd all 3 puzzles are solved here: [#1394](https://github.com/jcabi/jcabi-github/issues/1394), [#1399](https://github.com/jcabi/jcabi-github/issues/1399), [#1404](https://github.com/jcabi/jcabi-github/issues/1404).
